frontend/constants/routes.ts

```diff
@@ -41,3 +41,6 @@ export const REPORT_EDIT_ROUTE = (slug?: number): string =>
 
 export const PROFILE_ROUTE = (slug?: string): string =>
   slug ? `/user/${slug}` : '/user/[user]'
+
+export const CLUBS_HOME = 'http://pennclubs.com/'
+export const HUB_HOME = 'http://hub.provost.upenn.edu/'
```

Comment:
You should use the `https` versions of the sites instead of the `http` version.

Also grep the codebase for existing constants that don't use this new constants, and update them.